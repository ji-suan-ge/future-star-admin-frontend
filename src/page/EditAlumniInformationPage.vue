<template>
  <el-container>
    <el-main>
      <el-page-header @back="goBack()" content="修改校友信息"></el-page-header>
      <el-divider/>
      <el-card class="activity-card" shadow="always">
        <el-form class="activity-table" label-width="150px">
          <el-row>
            <el-col :span="8">
              <el-form-item label="姓名：">
                <el-input></el-input>
              </el-form-item>
            </el-col>
            <el-col :span="7">
              <el-form-item label="性别：">
                <el-radio label="男">男</el-radio>
                <el-radio label="女">女</el-radio>
              </el-form-item>
            </el-col>
            <el-col :span="8">
              <el-form-item label="生日：">
                <el-date-picker
                  type="date"
                  placeholder="选择日期">
                </el-date-picker>
              </el-form-item>
            </el-col>
          </el-row>
          <el-row>
            <el-col :span="8">
              <el-form-item label="电话：">
                <el-input></el-input>
              </el-form-item>
            </el-col>
            <el-col :span="8">
              <el-form-item label="微信：">
                <el-input></el-input>
              </el-form-item>
            </el-col>
            <el-col :span="8">
              <el-form-item label="邮箱：">
                <el-input></el-input>
              </el-form-item>
            </el-col>
          </el-row>
          <el-row>
            <el-col :span="8">
              <el-form-item label="所处城市：">
                <el-input></el-input>
              </el-form-item>
            </el-col>
            <el-col :span="8">
              <el-form-item label="最高学位：">
                <el-input></el-input>
              </el-form-item>
            </el-col>
            <el-col :span="8">
              <el-form-item label="毕业院校：">
                <el-input></el-input>
              </el-form-item>
            </el-col>
          </el-row>
          <el-row>
            <el-col :span="8">
              <el-form-item label="创业前所处公司：">
                <el-input></el-input>
              </el-form-item>
            </el-col>
            <el-col :span="8">
              <el-form-item label="创业前职位：">
                <el-input></el-input>
              </el-form-item>
            </el-col>
          </el-row>
          <el-row>
            <el-col :span="6">
              <el-form-item label="公众号：">
                <el-input></el-input>
              </el-form-item>
            </el-col>
            <el-col :span="6">
              <el-form-item label="成立时间：">
                <el-input></el-input>
              </el-form-item>
            </el-col>
            <el-col :span="6">
              <el-form-item label="公司所处城市：">
                <el-input></el-input>
              </el-form-item>
            </el-col>
            <el-col :span="6">
              <el-form-item label="员工数目：">
                <el-input></el-input>
              </el-form-item>
            </el-col>
          </el-row>
          <el-row>
          <el-row>
            <el-col span="8">
              <el-form-item label="公司：">
                <el-input></el-input>
              </el-form-item>
            </el-col>
            <el-col span="16">
              <el-form-item label="官网：">
                <el-input></el-input>
              </el-form-item>
            </el-col>
          </el-row>
            <el-col :span="8">
              <el-form-item label="所处职位：">
                <el-input></el-input>
              </el-form-item>
            </el-col>
          </el-row>
          <el-form-item label="公司简介：">
            <el-input
              type="textarea"
              :rows="2">
            </el-input>
          </el-form-item>
          <el-row>
            <el-col :span="6">
              <el-form-item label="运营数据：">
                <el-input></el-input>
              </el-form-item>
            </el-col>
            <el-col :span="6">
              <el-form-item label="收入规模：">
                <el-input></el-input>
              </el-form-item>
            </el-col>
            <el-col :span="6">
              <el-form-item label="融资情况：">
                <el-input></el-input>
              </el-form-item>
            </el-col>
            <el-col :span="6">
              <el-form-item label="当前估值：">
                <el-input></el-input>
              </el-form-item>
            </el-col>
          </el-row>
          <el-form-item>
            <el-button type="primary">确认修改</el-button>
            <el-button>取消</el-button>
          </el-form-item>
        </el-form>
      </el-card>
    </el-main>
  </el-container>
</template>

<script>
export default {
  name: 'EditAlumniInformationPage',
  methods: {
    goBack: function () {
      this.$router.go(-1)
    }
  }
}
</script>

<style scoped>
  .activity-card {
    width: 90%;
    margin: auto;
  }

  .activity-table {
    margin: 20px;
  }
</style>
