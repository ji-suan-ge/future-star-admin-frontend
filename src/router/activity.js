import CreateActivityPage from '../page/CreateActivityPage'
import ShowActivityParticipantsPage from '../page/ShowActivityParticipantsPage'
import EditExistingActivityPage from '../page/EditExistingActivityPage'
import ShowActivityInformationPage from '../page/ShowActivityInformationPage'

export default [
  {
    path: '/create-activity',
    name: 'CreateActivityPage',
    component: CreateActivityPage
  },
  {
    path: '/show-participants',
    name: 'ShowActivityParticipantsPage',
    component: ShowActivityParticipantsPage
  },
  {
    path: '/edit-activity',
    name: 'EditExistingActivityPage',
    component: EditExistingActivityPage
  },
  {
    path: '/show-activity-details',
    name: 'ShowActivityInformationPage',
    component: ShowActivityInformationPage
  }
]
