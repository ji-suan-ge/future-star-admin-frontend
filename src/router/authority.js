import AddAdministratorPage from '../page/authority-management/AddAdministratorPage'

export default [
  {
    path: '/add-administrator',
    name: 'AddAdministratorPage',
    component: AddAdministratorPage
  }
]
