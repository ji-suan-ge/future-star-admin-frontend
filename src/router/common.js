import LoginPage from '../page/LoginPage'
import HomePage from '../page/HomePage'
import homeChildren from './home-children'

export default [
  {
    path: '/',
    name: 'Root',
    redirect: '/login'
  },
  {
    path: '/login',
    name: 'LoginPage',
    component: LoginPage
  },
  {
    path: '/home',
    name: 'HomePage',
    component: HomePage,
    redirect: '/home/welcome',
    children: [
      ...homeChildren
    ]
  }
]
