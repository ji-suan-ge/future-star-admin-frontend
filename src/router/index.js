import Vue from 'vue'
import VueRouter from 'vue-router'
import LoginPage from '../page/LoginPage'
import HomePage from '../page/HomePage'
import EnrollmentManagementPage from '../page/EnrollmentManagementPage'
import CourseManagementPage from '../page/CourseManagementPage'
import ActivityManagementPage from '../page/ActivityManagementPage'
import AlumniManagementPage from '../page/AlumniManagementPage'
import AuthorityManagementPage from '../page/AuthorityManagementPage'
import CreateActivityPage from '../page/CreateActivityPage'
import AddAdministrator from '../page/AddAdministrator'
import WelcomePage from '../page/WelcomPage'

Vue.use(VueRouter)

const routes = [
  {
    path: '/',
    name: 'Root',
    redirect: '/login'
  },
  {
    path: '/login',
    name: 'LoginPage',
    component: LoginPage
  },
  {
    path: '/home',
    name: 'HomePage',
    component: HomePage,
    redirect: '/home/welcome',
    children: [
      {
        path: 'welcome',
        name: 'WelcomePage',
        component: WelcomePage
      },
      {
        path: 'enrollment-management',
        name: 'EnrollmentManagementPage',
        component: EnrollmentManagementPage
      },
      {
        path: 'course-management',
        name: 'CourseManagementPage',
        component: CourseManagementPage
      },
      {
        path: 'activity-management',
        name: 'ActivityManagementPage',
        component: ActivityManagementPage
      },
      {
        path: 'alumni-management',
        name: 'AlumniManagementPage',
        component: AlumniManagementPage
      },
      {
        path: 'authority-management',
        name: 'AuthorityManagementPage',
        component: AuthorityManagementPage
      },
      {
        path: 'add-administrator',
        name: 'AddAdministrator',
        component: AddAdministrator
      }
    ]
  },
  {
    path: '/create-activity',
    name: 'CreateActivityPage',
    component: CreateActivityPage
  }
]

const router = new VueRouter({
  routes
})

export default router
