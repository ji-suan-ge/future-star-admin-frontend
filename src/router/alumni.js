import ShowAlumniInformationPage from '../page/alumni-management/ShowAlumniInformationPage'
import EditAlumniInformationPage from '../page/alumni-management/EditAlumniInformationPage'
import ShowAlumniEditHistoryPage from '../page/alumni-management/ShowAlumniEditHistoryPage'
import ShowAlumniHistoryInformationPage from '../page/alumni-management/ShowAlumniHistoryInformationPage'

export default [
  {
    path: '/show-all-information',
    name: 'ShowAlumniInformationPage',
    component: ShowAlumniInformationPage
  },
  {
    path: '/edit-alumni-information',
    name: 'EditAlumniInformationPage',
    component: EditAlumniInformationPage
  },
  {
    path: '/show-edit-history',
    name: 'ShowAlumniEditHistoryPage',
    component: ShowAlumniEditHistoryPage
  },
  {
    path: '/show-history-information',
    name: 'ShowAlumniHistoryInformationPage',
    component: ShowAlumniHistoryInformationPage
  }
]
