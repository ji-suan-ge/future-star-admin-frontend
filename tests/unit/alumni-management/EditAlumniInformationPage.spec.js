import { createLocalVue, mount } from '@vue/test-utils'
import EditAlumniInformationPage from '../../../src/page/alumni-management/EditAlumniInformationPage'
import Vuex from 'vuex'
import ElementUI from 'element-ui'
import VueRouter from 'vue-router'
import store from '../../../src/store'
import router from '../../../src/router'

describe('EditAlumniInformationPage', () => {
  const localVue = createLocalVue()
  localVue.use(Vuex)
  localVue.use(ElementUI)
  localVue.use(VueRouter)
  const wrapper = mount(EditAlumniInformationPage, {
    store,
    router,
    localVue
  })
  test('快照测试', async () => {
    await expect(wrapper.html()).toMatchSnapshot()
  })
  test('是一个 Vue 实例', async () => {
    await expect(wrapper.isVueInstance()).toBeTruthy()
  })
  test('按钮', async () => {
    const header = wrapper.find('.el-page-header')
    await expect(header.props('content')).toBe('修改校友信息')
  })
})
