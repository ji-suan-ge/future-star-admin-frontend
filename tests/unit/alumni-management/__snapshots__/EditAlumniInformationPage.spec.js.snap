// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`EditAlumniInformationPage 快照测试 1`] = `
<section class="el-container">
  <main class="el-main">
    <div class="el-page-header">
      <div class="el-page-header__left"><i class="el-icon-back"></i>
        <div class="el-page-header__title">返回</div>
      </div>
      <div class="el-page-header__content">修改校友信息</div>
    </div>
    <div class="el-divider el-divider--horizontal">
      <!---->
    </div>
    <div class="el-card info-card is-always-shadow">
      <!---->
      <div class="el-card__body">
        <div class="el-row">
          <div class="avatar el-col el-col-24">
            <div class="block" style="margin: 30px auto 0px;">
              <div>
                <div tabindex="0" class="el-upload el-upload--text"><span class="el-avatar el-avatar--circle" style="height: 130px; width: 130px; line-height: 130px;"></span><input type="file" name="file" class="el-upload__input"></div>
                <ul class="el-upload-list el-upload-list--text"></ul>
              </div>
            </div>
          </div>
        </div>
        <form class="el-form info-table">
          <div class="line">
            <div class="el-divider el-divider--horizontal">
              <!---->
            </div>
          </div>
          <div class="description">- 个人信息 -</div>
          <div class="el-row" style="margin-top: 30px;">
            <div class="el-col el-col-5 el-col-offset-2"><i class="el-icon-user">&nbsp;姓名：</i>
              <div class="el-input el-input--size el-input-group el-input-group--prepend">
                <div class="el-input-group__prepend">名字:</div><input type="text" autocomplete="off" placeholder="请输入内容" maxlength="10" class="el-input__inner">
                <!----><span class="el-input__suffix"><span class="el-input__suffix-inner"><!----><!----><!----><span class="el-input__count"><span class="el-input__count-inner">
            0/10
          </span></span></span>
                <!----></span>
                <!---->
                <!---->
              </div>
            </div>
            <div class="el-col el-col-5 el-col-offset-2"><i class="el-icon-male">&nbsp;性别：</i>
              <div class="el-input el-input--medium el-input-group el-input-group--append el-input-group--prepend">
                <div class="el-input-group__prepend">性别:</div><input type="text" autocomplete="off" placeholder="请输入内容" maxlength="1" class="el-input__inner">
                <!----><span class="el-input__suffix" style="transform: translateX(-0px);"><span class="el-input__suffix-inner"><!----><!----><!----><span class="el-input__count"><span class="el-input__count-inner">
            0/1
          </span></span></span>
                <!----></span>
                <div class="el-input-group__append">
                  <div class="el-select">
                    <!---->
                    <div class="el-input el-input--suffix">
                      <!----><input type="text" readonly="readonly" autocomplete="off" placeholder="请选择" class="el-input__inner">
                      <!----><span class="el-input__suffix"><span class="el-input__suffix-inner"><i class="el-select__caret el-input__icon el-icon-arrow-up"></i><!----><!----><!----><!----><!----></span>
                      <!----></span>
                      <!---->
                      <!---->
                    </div>
                    <div class="el-select-dropdown el-popper" style="display: none;">
                      <div class="el-scrollbar" style="">
                        <div class="el-select-dropdown__wrap el-scrollbar__wrap el-scrollbar__wrap--hidden-default">
                          <ul class="el-scrollbar__view el-select-dropdown__list">
                            <!---->
                            <li class="el-select-dropdown__item"><span>男</span></li>
                            <li class="el-select-dropdown__item"><span>女</span></li>
                          </ul>
                        </div>
                        <div class="el-scrollbar__bar is-horizontal">
                          <div class="el-scrollbar__thumb" style="transform: translateX(0%); webkit-transform: translateX(0%);"></div>
                        </div>
                        <div class="el-scrollbar__bar is-vertical">
                          <div class="el-scrollbar__thumb" style="transform: translateY(0%); webkit-transform: translateY(0%);"></div>
                        </div>
                      </div>
                      <!---->
                    </div>
                  </div>
                </div>
                <!---->
              </div>
            </div>
            <div class="el-col el-col-5 el-col-offset-2"><i class="el-icon-timer">&nbsp;生日：</i>
              <div class="el-input el-input--medium is-disabled el-input-group el-input-group--prepend el-input--suffix">
                <div class="el-input-group__prepend">日期:</div><input type="text" disabled="disabled" autocomplete="off" placeholder="请输入内容" maxlength="10" class="el-input__inner">
                <!----><span class="el-input__suffix"><span class="el-input__suffix-inner"><i class="el-input__icon el-icon-date"></i><!----><!----><!----></span>
                <!----></span>
                <!---->
                <!---->
              </div>
            </div>
          </div>
          <div class="el-row" style="margin-top: 30px;">
            <div class="el-col el-col-5 el-col-offset-2"><i class="el-icon-mobile-phone">&nbsp;电话：</i>
              <div class="el-input el-input--medium el-input-group el-input-group--prepend">
                <div class="el-input-group__prepend">电话号码:</div><input type="text" autocomplete="off" placeholder="请输入内容" maxlength="11" class="el-input__inner">
                <!----><span class="el-input__suffix"><span class="el-input__suffix-inner"><!----><!----><!----><span class="el-input__count"><span class="el-input__count-inner">
            0/11
          </span></span></span>
                <!----></span>
                <!---->
                <!---->
              </div>
            </div>
            <div class="el-col el-col-5 el-col-offset-2"><i class="el-icon-document">&nbsp;最高学历：</i>
              <div class="el-input el-input--medium el-input-group el-input-group--append el-input-group--prepend">
                <div class="el-input-group__prepend">学历:</div><input type="text" autocomplete="off" placeholder="请输入内容" maxlength="10" class="el-input__inner">
                <!----><span class="el-input__suffix" style="transform: translateX(-0px);"><span class="el-input__suffix-inner"><!----><!----><!----><span class="el-input__count"><span class="el-input__count-inner">
            0/10
          </span></span></span>
                <!----></span>
                <div class="el-input-group__append">
                  <div class="el-select">
                    <!---->
                    <div class="el-input el-input--suffix">
                      <!----><input type="text" readonly="readonly" autocomplete="off" placeholder="请选择" class="el-input__inner">
                      <!----><span class="el-input__suffix"><span class="el-input__suffix-inner"><i class="el-select__caret el-input__icon el-icon-arrow-up"></i><!----><!----><!----><!----><!----></span>
                      <!----></span>
                      <!---->
                      <!---->
                    </div>
                    <div class="el-select-dropdown el-popper" style="display: none;">
                      <div class="el-scrollbar" style="">
                        <div class="el-select-dropdown__wrap el-scrollbar__wrap el-scrollbar__wrap--hidden-default">
                          <ul class="el-scrollbar__view el-select-dropdown__list">
                            <!---->
                            <li class="el-select-dropdown__item"><span>博士</span></li>
                            <li class="el-select-dropdown__item"><span>研究生</span></li>
                            <li class="el-select-dropdown__item"><span>本科</span></li>
                            <li class="el-select-dropdown__item"><span>专科</span></li>
                            <li class="el-select-dropdown__item"><span>其他</span></li>
                          </ul>
                        </div>
                        <div class="el-scrollbar__bar is-horizontal">
                          <div class="el-scrollbar__thumb" style="transform: translateX(0%); webkit-transform: translateX(0%);"></div>
                        </div>
                        <div class="el-scrollbar__bar is-vertical">
                          <div class="el-scrollbar__thumb" style="transform: translateY(0%); webkit-transform: translateY(0%);"></div>
                        </div>
                      </div>
                      <!---->
                    </div>
                  </div>
                </div>
                <!---->
              </div>
            </div>
            <div class="el-col el-col-5 el-col-offset-2"><i class="el-icon-school">&nbsp;毕业院校：</i>
              <div class="el-input el-input--medium el-input-group el-input-group--prepend">
                <div class="el-input-group__prepend">学校名:</div><input type="text" autocomplete="off" placeholder="请输入内容" maxlength="10" class="el-input__inner">
                <!----><span class="el-input__suffix"><span class="el-input__suffix-inner"><!----><!----><!----><span class="el-input__count"><span class="el-input__count-inner">
            0/10
          </span></span></span>
                <!----></span>
                <!---->
                <!---->
              </div>
            </div>
          </div>
          <div class="el-row" style="margin-top: 30px;">
            <div class="el-col el-col-5 el-col-offset-2"><i class="el-icon-location-information">&nbsp;城市：</i>
              <div class="el-input el-input--medium el-input-group el-input-group--prepend">
                <div class="el-input-group__prepend">城市名:</div><input type="text" autocomplete="off" placeholder="请输入内容" maxlength="10" class="el-input__inner">
                <!----><span class="el-input__suffix"><span class="el-input__suffix-inner"><!----><!----><!----><span class="el-input__count"><span class="el-input__count-inner">
            0/10
          </span></span></span>
                <!----></span>
                <!---->
                <!---->
              </div>
            </div>
            <div class="el-col el-col-5 el-col-offset-2"><i class="el-icon-chat-dot-round">&nbsp;微信：</i>
              <div class="el-input el-input--medium el-input-group el-input-group--prepend">
                <div class="el-input-group__prepend">微信号:</div><input type="text" autocomplete="off" placeholder="请输入内容" maxlength="20" class="el-input__inner">
                <!----><span class="el-input__suffix"><span class="el-input__suffix-inner"><!----><!----><!----><span class="el-input__count"><span class="el-input__count-inner">
            0/20
          </span></span></span>
                <!----></span>
                <!---->
                <!---->
              </div>
            </div>
            <div class="el-col el-col-5 el-col-offset-2"><i class="el-icon-edit-outline">&nbsp;邮箱：</i>
              <div class="el-input el-input--medium el-input-group el-input-group--prepend">
                <div class="el-input-group__prepend">邮箱号:</div><input type="text" autocomplete="off" placeholder="请输入内容" maxlength="30" class="el-input__inner">
                <!----><span class="el-input__suffix"><span class="el-input__suffix-inner"><!----><!----><!----><span class="el-input__count"><span class="el-input__count-inner">
            0/30
          </span></span></span>
                <!----></span>
                <!---->
                <!---->
              </div>
            </div>
          </div>
          <div class="line">
            <div class="el-divider el-divider--horizontal">
              <!---->
            </div>
          </div>
          <div class="el-row" style="margin-bottom: 40px;">
            <div class="el-col el-col-24">
              <div class="description">- 创业前所在公司 -</div>
            </div>
          </div>
          <div class="el-row" style="margin-top: 30px;">
            <div class="el-col el-col-8 el-col-offset-3"><i class="el-icon-office-building">&nbsp;公司名称：</i>
              <div class="el-input el-input--medium el-input-group el-input-group--prepend">
                <div class="el-input-group__prepend">公司名:</div><input type="text" autocomplete="off" placeholder="请输入内容" maxlength="20" class="el-input__inner">
                <!----><span class="el-input__suffix"><span class="el-input__suffix-inner"><!----><!----><!----><span class="el-input__count"><span class="el-input__count-inner">
            0/20
          </span></span></span>
                <!----></span>
                <!---->
                <!---->
              </div>
            </div>
            <div class="el-col el-col-8 el-col-offset-3"><i class="el-icon-trophy">&nbsp;前职位：</i>
              <div class="el-input el-input--medium el-input-group el-input-group--append el-input-group--prepend">
                <div class="el-input-group__prepend">职位:</div><input type="text" autocomplete="off" placeholder="请输入内容" maxlength="20" class="el-input__inner">
                <!----><span class="el-input__suffix" style="transform: translateX(-0px);"><span class="el-input__suffix-inner"><!----><!----><!----><span class="el-input__count"><span class="el-input__count-inner">
            0/20
          </span></span></span>
                <!----></span>
                <div class="el-input-group__append">
                  <div class="el-select">
                    <!---->
                    <div class="el-input el-input--suffix">
                      <!----><input type="text" readonly="readonly" autocomplete="off" placeholder="可选择" class="el-input__inner">
                      <!----><span class="el-input__suffix"><span class="el-input__suffix-inner"><i class="el-select__caret el-input__icon el-icon-arrow-up"></i><!----><!----><!----><!----><!----></span>
                      <!----></span>
                      <!---->
                      <!---->
                    </div>
                    <div class="el-select-dropdown el-popper" style="display: none;">
                      <div class="el-scrollbar" style="">
                        <div class="el-select-dropdown__wrap el-scrollbar__wrap el-scrollbar__wrap--hidden-default">
                          <ul class="el-scrollbar__view el-select-dropdown__list">
                            <!---->
                            <li class="el-select-dropdown__item"><span>创始人</span></li>
                            <li class="el-select-dropdown__item"><span>联合创始人</span></li>
                            <li class="el-select-dropdown__item"><span>董事长</span></li>
                            <li class="el-select-dropdown__item"><span>首席执行官</span></li>
                            <li class="el-select-dropdown__item"><span>其他</span></li>
                          </ul>
                        </div>
                        <div class="el-scrollbar__bar is-horizontal">
                          <div class="el-scrollbar__thumb" style="transform: translateX(0%); webkit-transform: translateX(0%);"></div>
                        </div>
                        <div class="el-scrollbar__bar is-vertical">
                          <div class="el-scrollbar__thumb" style="transform: translateY(0%); webkit-transform: translateY(0%);"></div>
                        </div>
                      </div>
                      <!---->
                    </div>
                  </div>
                </div>
                <!---->
              </div>
            </div>
          </div>
          <div class="line">
            <div class="el-divider el-divider--horizontal">
              <!---->
            </div>
          </div>
          <div class="el-row" style="margin-bottom: 40px;">
            <div class="el-col el-col-24">
              <div class="description">- 当前公司 -</div>
            </div>
          </div>
          <div class="el-row" style="margin-top: 30px;">
            <div class="el-col el-col-5 el-col-offset-2"><i class="el-icon-star-off">&nbsp;名称：</i>
              <div class="el-input el-input--medium el-input-group el-input-group--prepend">
                <div class="el-input-group__prepend">公司名:</div><input type="text" autocomplete="off" placeholder="请输入内容" maxlength="20" class="el-input__inner">
                <!----><span class="el-input__suffix"><span class="el-input__suffix-inner"><!----><!----><!----><span class="el-input__count"><span class="el-input__count-inner">
            0/20
          </span></span></span>
                <!----></span>
                <!---->
                <!---->
              </div>
            </div>
            <div class="el-col el-col-5 el-col-offset-2"><i class="el-icon-house">&nbsp;官网：</i>
              <div class="el-input el-input--medium el-input-group el-input-group--prepend">
                <div class="el-input-group__prepend">Http://</div><input type="text" autocomplete="off" placeholder="请输入内容" maxlength="20" class="el-input__inner">
                <!----><span class="el-input__suffix"><span class="el-input__suffix-inner"><!----><!----><!----><span class="el-input__count"><span class="el-input__count-inner">
            0/20
          </span></span></span>
                <!----></span>
                <!---->
                <!---->
              </div>
            </div>
            <div class="el-col el-col-5 el-col-offset-2"><i class="el-icon-discount">&nbsp;公众号：</i>
              <div class="el-input el-input--medium el-input-group el-input-group--prepend">
                <div class="el-input-group__prepend">公众号名称:</div><input type="text" autocomplete="off" placeholder="请输入内容" maxlength="20" class="el-input__inner">
                <!----><span class="el-input__suffix"><span class="el-input__suffix-inner"><!----><!----><!----><span class="el-input__count"><span class="el-input__count-inner">
            0/20
          </span></span></span>
                <!----></span>
                <!---->
                <!---->
              </div>
            </div>
          </div>
          <div class="el-row" style="margin-top: 30px;">
            <div class="el-col el-col-5 el-col-offset-2"><i class="el-icon-alarm-clock">&nbsp;成立时间：</i>
              <div class="el-input el-input--medium is-disabled el-input-group el-input-group--prepend">
                <div class="el-input-group__prepend">日期:</div><input type="text" disabled="disabled" autocomplete="off" placeholder="请输入内容" maxlength="20" class="el-input__inner">
                <!---->
                <!---->
                <!---->
                <!---->
              </div>
            </div>
            <div class="el-col el-col-5 el-col-offset-2"><i class="el-icon-place">&nbsp;所在城市：</i>
              <div class="el-input el-input--medium el-input-group el-input-group--prepend">
                <div class="el-input-group__prepend">城市:</div><input type="text" autocomplete="off" placeholder="请输入内容" maxlength="20" class="el-input__inner">
                <!----><span class="el-input__suffix"><span class="el-input__suffix-inner"><!----><!----><!----><span class="el-input__count"><span class="el-input__count-inner">
            0/20
          </span></span></span>
                <!----></span>
                <!---->
                <!---->
              </div>
            </div>
            <div class="el-col el-col-5 el-col-offset-2"><i class="el-icon-data-analysis">&nbsp;总人数：</i>
              <div class="el-input el-input--medium el-input-group el-input-group--prepend">
                <div class="el-input-group__prepend">人数(人):</div><input type="text" autocomplete="off" placeholder="请输入内容" maxlength="20" class="el-input__inner">
                <!----><span class="el-input__suffix"><span class="el-input__suffix-inner"><!----><!----><!----><span class="el-input__count"><span class="el-input__count-inner">
            0/20
          </span></span></span>
                <!----></span>
                <!---->
                <!---->
              </div>
            </div>
          </div>
          <div class="el-row" style="margin-top: 30px;">
            <div class="el-col el-col-5 el-col-offset-2"><i class="el-icon-trophy">&nbsp;职位：</i>
              <div class="el-input el-input--medium el-input-group el-input-group--append el-input-group--prepend">
                <div class="el-input-group__prepend">职称:</div><input type="text" autocomplete="off" placeholder="请输入内容" maxlength="20" class="el-input__inner">
                <!----><span class="el-input__suffix" style="transform: translateX(-0px);"><span class="el-input__suffix-inner"><!----><!----><!----><span class="el-input__count"><span class="el-input__count-inner">
            0/20
          </span></span></span>
                <!----></span>
                <div class="el-input-group__append">
                  <div class="el-select">
                    <!---->
                    <div class="el-input el-input--suffix">
                      <!----><input type="text" readonly="readonly" autocomplete="off" placeholder="可选择" class="el-input__inner">
                      <!----><span class="el-input__suffix"><span class="el-input__suffix-inner"><i class="el-select__caret el-input__icon el-icon-arrow-up"></i><!----><!----><!----><!----><!----></span>
                      <!----></span>
                      <!---->
                      <!---->
                    </div>
                    <div class="el-select-dropdown el-popper" style="display: none;">
                      <div class="el-scrollbar" style="">
                        <div class="el-select-dropdown__wrap el-scrollbar__wrap el-scrollbar__wrap--hidden-default">
                          <ul class="el-scrollbar__view el-select-dropdown__list">
                            <!---->
                            <li class="el-select-dropdown__item"><span>创始人</span></li>
                            <li class="el-select-dropdown__item"><span>联合创始人</span></li>
                            <li class="el-select-dropdown__item"><span>董事长</span></li>
                            <li class="el-select-dropdown__item"><span>首席执行官</span></li>
                            <li class="el-select-dropdown__item"><span>其他</span></li>
                          </ul>
                        </div>
                        <div class="el-scrollbar__bar is-horizontal">
                          <div class="el-scrollbar__thumb" style="transform: translateX(0%); webkit-transform: translateX(0%);"></div>
                        </div>
                        <div class="el-scrollbar__bar is-vertical">
                          <div class="el-scrollbar__thumb" style="transform: translateY(0%); webkit-transform: translateY(0%);"></div>
                        </div>
                      </div>
                      <!---->
                    </div>
                  </div>
                </div>
                <!---->
              </div>
            </div>
            <div class="el-col el-col-5 el-col-offset-2"><i class="el-icon-news">&nbsp;融资情况：</i>
              <div class="el-input el-input--medium el-input-group el-input-group--append el-input-group--prepend">
                <div class="el-input-group__prepend">轮次:</div><input type="text" autocomplete="off" placeholder="请输入内容" maxlength="20" class="el-input__inner">
                <!----><span class="el-input__suffix" style="transform: translateX(-0px);"><span class="el-input__suffix-inner"><!----><!----><!----><span class="el-input__count"><span class="el-input__count-inner">
            0/20
          </span></span></span>
                <!----></span>
                <div class="el-input-group__append">
                  <div class="el-select">
                    <!---->
                    <div class="el-input el-input--suffix">
                      <!----><input type="text" readonly="readonly" autocomplete="off" placeholder="可选择" class="el-input__inner">
                      <!----><span class="el-input__suffix"><span class="el-input__suffix-inner"><i class="el-select__caret el-input__icon el-icon-arrow-up"></i><!----><!----><!----><!----><!----></span>
                      <!----></span>
                      <!---->
                      <!---->
                    </div>
                    <div class="el-select-dropdown el-popper" style="display: none;">
                      <div class="el-scrollbar" style="">
                        <div class="el-select-dropdown__wrap el-scrollbar__wrap el-scrollbar__wrap--hidden-default">
                          <ul class="el-scrollbar__view el-select-dropdown__list">
                            <!---->
                            <li class="el-select-dropdown__item"><span>种子轮</span></li>
                            <li class="el-select-dropdown__item"><span>天使轮</span></li>
                            <li class="el-select-dropdown__item"><span>A轮</span></li>
                            <li class="el-select-dropdown__item"><span>B轮</span></li>
                            <li class="el-select-dropdown__item"><span>C轮</span></li>
                            <li class="el-select-dropdown__item"><span>D轮</span></li>
                            <li class="el-select-dropdown__item"><span>其他</span></li>
                          </ul>
                        </div>
                        <div class="el-scrollbar__bar is-horizontal">
                          <div class="el-scrollbar__thumb" style="transform: translateX(0%); webkit-transform: translateX(0%);"></div>
                        </div>
                        <div class="el-scrollbar__bar is-vertical">
                          <div class="el-scrollbar__thumb" style="transform: translateY(0%); webkit-transform: translateY(0%);"></div>
                        </div>
                      </div>
                      <!---->
                    </div>
                  </div>
                </div>
                <!---->
              </div>
            </div>
            <div class="el-col el-col-5 el-col-offset-2"><i class="el-icon-coin">&nbsp;当前估值：</i>
              <div class="el-input el-input--medium el-input-group el-input-group--prepend">
                <div class="el-input-group__prepend">估值(¥):</div><input type="text" autocomplete="off" placeholder="请输入内容" maxlength="20" class="el-input__inner">
                <!----><span class="el-input__suffix"><span class="el-input__suffix-inner"><!----><!----><!----><span class="el-input__count"><span class="el-input__count-inner">
            0/20
          </span></span></span>
                <!----></span>
                <!---->
                <!---->
              </div>
            </div>
          </div>
          <div class="el-row" style="margin-top: 30px;">
            <div class="el-col el-col-21 el-col-offset-2"><i class="el-icon-money">&nbsp;收入规模：</i>
              <div class="el-textarea"><textarea autocomplete="off" placeholder="请输入内容" maxlength="100" class="el-textarea__inner" style="min-height: -8px; height: -16px;"></textarea><span class="el-input__count">0/100</span></div>
            </div>
          </div>
          <div class="el-row" style="margin-top: 30px;">
            <div class="el-col el-col-21 el-col-offset-2"><i class="el-icon-shopping-cart-full">&nbsp;运营数据：</i>
              <div class="el-textarea"><textarea autocomplete="off" placeholder="请输入内容" maxlength="100" class="el-textarea__inner" style="min-height: -8px; height: -16px;"></textarea><span class="el-input__count">0/100</span></div>
            </div>
          </div>
          <div class="el-row" style="margin-top: 30px;">
            <div class="el-col el-col-21 el-col-offset-2"><i class="el-icon-reading">&nbsp;公司简介：</i>
              <div class="el-textarea"><textarea autocomplete="off" placeholder="请输入内容" maxlength="400" class="el-textarea__inner" style="min-height: -8px; height: -32px;"></textarea><span class="el-input__count">0/400</span></div>
            </div>
          </div>
        </form>
        <div class="el-row">
          <div class="button el-col el-col-24"><button type="button" class="el-button el-button--primary">
              <!---->
              <!----><span>确认修改</span></button> <button type="button" class="el-button el-button--default">
              <!---->
              <!----><span>取消</span></button></div>
        </div>
      </div>
    </div>
  </main>
</section>
`;
